### PR TITLE
doc: Modified uftrace-script.md

### PR DESCRIPTION
**Pull request by gonapps**
Add some sentenses about LuaJIT to uftrace-script.md

Signed-off-by: Byeonggon Lee <gonny952@gmail.com>

### CODE REVIEW

File: doc/uftrace-script.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ SCRIPT OPTIONS
 -S *SCRIPT_PATH*, \--script=*SCRIPT_PATH*
 :   Run a given script to do additional work at the entry and exit of function
     while processing recorded trace data.
-    The type of script is detected by the postfix such as '.py' for python.
+    The type of script is detected by the file extension.
+    for example '.py' for python and '.lua' for luajit.
```
Comment (honggyukim) [inline doc/uftrace-script.md]: Please use a capital letter.
"for example" -> "For example,"

Comment (gonapps) [reply doc/uftrace-script.md]: Thanks, I fixed.

File: doc/uftrace-script.md [hunk 2]
```diff
@@ -94,7 +95,7 @@ COMMON ANALYSIS OPTIONS
 SCRIPT EXECUTION
 ================
 The uftrace tool supports script execution for each function entry and exit.
-The supported script is only Python 2.7 as of now.
+The supported script types are Python 2.7 and LuaJIT as of now.
```
Comment (namhyung) [inline doc/uftrace-script.md]: To be precise, it should be Lua 5.1 right?

Comment (gonapps) [reply doc/uftrace-script.md]: Yes. should I change to lua5.1?

Comment (namhyung) [reply doc/uftrace-script.md]: Please.

Comment (gonapps) [reply doc/uftrace-script.md]: Updated to Lua 5.1

**Comment by honggyukim** [comment]
@gonapps Can you update Korean man page as well?

**Comment by gonapps** [comment]
@honggyukim okay I'll update

**Comment by honggyukim** [comment]
@gonapps Could you please update?

**Comment by honggyukim** [comment]
@namhyung Hmm.. why don't we just merge this? It may take some time to get feedback from @gonapps.

**Comment by gonapps** [comment]
@honggyukim Sorry for the late response, I also updated korean doc

**Comment by honggyukim** [comment]
@gonapps You don't have to say sorry anyway. Thanks very much for the update!

**Pull request merged**